inspire/base/static/js/search/search_results.js

```diff
@@ -95,6 +94,7 @@ define(
       }
 
       this.onExportSelectAll = function(ev) {
+        var minimum_page_limit = $("#select-numpages option:selected").text();
```

Comment:
No, but it works like this var minimum_page_limit = $("#select-numpages
option:selected").val(); . Should we keep .text() or .val() ?


2015-11-24 11:58 GMT+01:00 Javier Martin Montull <notifications@github.com>:

> In inspire/base/static/js/search/search_results.js
> <https://github.com/inspirehep/inspire-next/pull/552#discussion_r45720998>
> :
>
> > @@ -95,6 +94,7 @@ define(
> >        }
> >
> >        this.onExportSelectAll = function(ev) {
> > +        var minimum_page_limit = $("#select-numpages option:selected").text();
>
> As it is a dropdown, doesn't it work with:
>
> var minimum_page_limit = $("#select-numpages").val();
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/552/files#r45720998>.
>